test/test_utils/test_reporting.py

```diff
@@ -43,8 +43,15 @@ class TestReportGenerator:
     Class to generate test report files
     """
 
-    ALLOWED_SINGLE_GPU_TESTS = ("telemetry", "test_framework_and_cuda_version_gpu", "test_curand_gpu",
-                                "test_dependency_check_gpu")
+    ALLOWED_SINGLE_GPU_TESTS = (
+        "telemetry",
+        "test_framework_and_cuda_version_gpu",
+        "test_curand_gpu",
+        "test_dependency_check_gpu",
+        "test_tensorflow_addons_gpu",
+        "test_tensorflow_standalone_gpu",
+        "test_tensorflow_tensorboard_gpu",
```

Comment:
@sboshin could you sign off that these tests are allowed to be single gpu?

Reply:
test_tensorflow_tensorboard_gpu should be multi-gpu.
Everything else looks fine to me.

Reply:
Thanks! Will make the change.